config/settings/base.py

```diff
@@ -412,11 +412,13 @@
     SPRINT_STATUS_IN_DEVELOPMENT,
 }
 # String for overriding how much time will be needed for an epic management per sprint.
-SPRINT_EPIC_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d+) hours per sprint for epic management"
+SPRINT_EPIC_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d*)\.?(\d*) hours per sprint for epic management"
 # String for overriding how much time will be needed for a recurring task per sprint.
-SPRINT_RECURRING_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d+) hours per sprint for this task"
+SPRINT_RECURRING_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d*)\.?(\d*) hours per sprint for this task"
 # String for overriding how much time will be needed for the task's review.
-SPRINT_REVIEW_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d+) hours for reviewing this task"
+SPRINT_REVIEW_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d*)\.?(\d*) hours for reviewing this task"
+# String for overriding how much time will be needed for the task's review.
+SPRINT_REVIEW_REMAINING_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d*)\.?(\d*) hours for the remaining review of this task"
```

Comment:
We could use `SPRINT_REVIEW_DIRECTIVE` there, but many tickets in the `External Review`/`Merged` don't require more time from the reviewer, so they would need to remember about removing the original value of `SPRINT_REVIEW_DIRECTIVE` from the description. It would also mean that we are not keeping the full ticket's history, as Jira doesn't retain the previous version of the description/comment after we edit it.

Reply:
In that case the person can just leave behind the previous review comment, adding a sprint number or date and add a new one after it. After all, if a ticket still needs review time a third time, we can't have another directive for the third/fourth review. I'm also not sure how much value there is in retaining the history of how much review time is left.

Reply:
@xitij2000, that sounds perfectly reasonable :+1: 
I posted this on the forum thread, as I wasn't the originator of this idea.

Reply:
@xitij2000, done. It doesn't make sense to have redundant directives.